hw12_Zhurylo/js/index.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener('DOMContentLoaded', function () {
+    const array = []
+    const arrayLength = Number(prompt('Enter array lenth'))
+    console.log(arrayLength)
+    for (let i = 1; i <= arrayLength; i++) {
+        array.push(prompt(`Enter array element ${i}`))
+    }
+    console.log(array)
+    console.log(array.sort())
```

Comment:
This `sort` will sort like a string. TO see how it works try [1, 2, 3, 12, 21, 13, 111]